tests/classiceditorui.js

```diff
@@ -13,7 +13,7 @@ import ClassicEditorUIView from '../src/classiceditoruiview';
 import ClassicTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/classictesteditor';
 
 import FocusTracker from '@ckeditor/ckeditor5-utils/src/focustracker';
-import { keyCodes } from '@ckeditor/ckeditor5-utils/src/keyboard';
+import * as enableToolbarKeyboardFocus from '@ckeditor/ckeditor5-ui/src/toolbar/enabletoolbarkeyboardfocus';
```

Comment:
I have bad feelings when I see such a use of ES6 modules. For me, this goes beyond what's safe, taken that the entire spec of ES6 modules doesn't define much. We're basing here on the way how they are transpiled by Babel which may be a subject of change. Besides, I remember people warning about `import *` so it'd better to rewrite this test without stubbing on modules.